src/jsx.d.ts

```diff
@@ -1613,6 +1613,9 @@ export namespace JSXInternal {
 
 	interface ButtonHTMLAttributes<T extends EventTarget = HTMLButtonElement>
 		extends HTMLAttributes<T> {
+		command?: Signalish<string | undefined>;
+		commandfor?: Signalist<string | undefined>;
+		commandFor?: Signalist<string | undefined>;
```

Comment:
There's a .commandForElement property that you can set to an element, it would be good to support this in Preact but popoverTargetElement also doesn't seem to be handled so will omit that for now.